entries/f/freee.co.jp.json

```
@@ -0,0 +1,12 @@
+{
+    "freee.co.jp": {
```

freee seems to be the official name of the service.
```suggestion
    "freee": {
```